k8s/sonarqube/README.md

```diff
@@ -0,0 +1,439 @@
+# Overview
+
+SonarQube is an open source platform to perform automatic reviews with static analysis of code to detect bugs,
+code smells and security vulnerabilities on 25+ programming languages including Java, C#, JavaScript, TypeScript,
+C/C++, COBOL and more. SonarQube is the only product on the market that supports a leak approach as a practice to code quality.
+
+For more information on SonarQube, see the [SonarQube website](https://www.sonarqube.org/).
+
+## About Google Click to Deploy
+
+Popular open source software stacks on Kubernetes packaged by Google and made available in Google Cloud Marketplace.
+
+## Design
+
+![Architecture diagram](resources/sonarqube-k8s-app-architecture.png)
+
+
+### SonarQube application contains:
```

Comment:
```suggestion
### SonarQube application contains:

```

https://github.com/google/styleguide/blob/gh-pages/docguide/style.md#add-spacing-to-headings